bpfd-operator/controllers/bpfd-operator/common.go

```diff
@@ -78,14 +80,14 @@ func reconcileBpfProgram(ctx context.Context, rec ProgramReconciler, prog client
 	opts := []client.ListOption{client.MatchingLabels{internal.BpfProgramOwnerLabel: progName}}
 
 	if err := r.List(ctx, bpfPrograms, opts...); err != nil {
-		r.Logger.Error(err, "failed to get freshPrograms for full reconcile")
+		r.Logger.V(1).Error(err, "failed to get freshPrograms for full reconcile")
 		return ctrl.Result{}, nil
 	}
 
 	// List all nodes since a bpfprogram object will always be created for each
 	nodes := &corev1.NodeList{}
 	if err := r.List(ctx, nodes, &client.ListOptions{}); err != nil {
-		r.Logger.Error(err, "failed getting nodes for full reconcile")
+		r.Logger.V(1).Error(err, "failed getting nodes for full reconcile")
```

Comment:
This is the only place where we see this error, I don't think you should bump these to V(1)

Reply:
fixed